docs/tutorials/ipfs-gallery/index.md

```diff
@@ -0,0 +1,586 @@
+In this tutorial we will create an app to create and share an image gallery using IPFS.
+
+To complete the tutorial, you will need the [Agregore Web](https://agregore.mauve.moe/) browser. Agregore supports several distributed protocols like IPFS and allows you to view and create the distributed web! You can find the latest version [here](https://github.com/AgregoreWeb/agregore-browser/releases/latest).
+
+We will be using an Integrated Development Environment developed in a previous tutorial to write the code for this tutorial. This allows you to create and update your site with nothing other than the Agregore Web browser!
+
+# Requirements
+
+We want to create an image sharing app where a user can add their own images, create a gallery with those images and share them with another user.
+
+The final application should be usable by anyone using Agregore Web and not require any deep technical knowledge.
+
+# Get started
+
+To get started, go to [this page](https://agregore.mauve.moe/docs/examples/browser-devenv-v3/) and click on **Start**. This will create a copy of the IDE for you to use during this tutorial.
+
+# Image layout
+
+We should decide how we want to layout our images. For inspiration we can look at how other popular photo sharing websites layout their images. I personally like the way that Flickr handles the layout. This image show the layout we are aiming for:
+
+![](ipfs-gallery-layout.png)
```

Comment:
It'd be good to check that we have alt text for any images for screen readers and network issues.

Reply:
Done